src/cli-generate-cross-account-api-keys.ts

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env node
+
+import yargs from 'yargs-parser';
+import colors from 'colors';
+import { writeFileSync } from 'fs';
+import keyBy from 'lodash/keyBy';
+import { ScopeName, TRANSCEND_SCOPES } from '@transcend-io/privacy-types';
+
+import { logger } from './logger';
+import { generateCrossAccountApiKeys } from './api-keys';
+
+const SCOPES_BY_TITLE = keyBy(
+  Object.entries(TRANSCEND_SCOPES).map(([name, value]) => ({
+    ...value,
+    name,
+  })),
+  'title',
+);
+const SCOPE_TITLES = Object.keys(SCOPES_BY_TITLE);
+
+/**
+ * Create API keys with the same set of scopes across multiple Transcend instances
+ *
+ * Requires the username and password of user with access to multiple instances.
+ *
+ * Dev Usage:
+ * yarn ts-node ./src/cli-generate-cross-account-api-keys.ts --email=test@transcend.io --email=NE8pGp$s8Gm4Mzb5 \
+ *   --apiKeyTitle="My Test Key" --scopes="View Email Templates,View Data Map" --file=./secrets.json
+ *
+ * Standard usage:
+ * yarn tr-generate-api-keys --email=test@transcend.io --password=NE8pGp$s8Gm4Mzb5 \
+ *   --apiKeyTitle="My Test Key" --scopes="View Email Templates,View Data Map" --file=./secrets.json
+ */
+async function main(): Promise<void> {
+  // Parse command line arguments
+  const {
+    transcendUrl = 'https://api.transcend.io',
+    file,
+    email,
+    password,
+    apiKeyTitle,
+    scopes,
+    deleteExistingApiKey = 'true',
+    createNewApiKey = 'true',
+  } = yargs(process.argv.slice(2)) as { [k in string]: string };
```

Comment:
Maybe we can add an option to limit what accounts they may wanna create keys in? Could see this being useful once we start having sub-teams manage their own set of accounts, rather than one team managing all of them

Reply:
ya ive got a ticket for that, on the same page! my plan was to filter on `parentOrganizationId` but it wasn't exposed in the GraphQL endpoints. i put up a separate PR to expose it, just waiting on the deploy to go out

Reply:
Sounds good